src/uniprotkb/config/__tests__/UniProtKBColumnConfiguration.spec.tsx

```diff
@@ -32,6 +34,13 @@ describe('UniProtKBColumnConfiguration component', () => {
         <>{column.render(uniProtKbConverter(data))}</>
       );
       expect(asFragment()).toMatchSnapshot(key);
+      const showMore =
+        screen.queryByTitle('Show more') ||
+        screen.queryByTestId('expandable-message');
+      if (showMore) {
+        showMore.click();
+        expect(asFragment()).toMatchSnapshot(`${key}-expanded`);
+      }
```

Comment:
Nice addition to the tests. Would you mind adding this pattern to the other namespaces' column renderer tests as part of this PR? Not sure that it applies to many of the columns of the other namespaces, but some do and some might in the future, so it's good to have it part of the tests already.